packages/auth/e2e/auth.e2e.js

```diff
@@ -1107,4 +1107,30 @@ describe('auth()', () => {
       should.not.exist(resetKeychain);
     });
   });
+
+  describe('sendEmailVerification()', () => {
+    it('should reject if dynamic links has not been configured with action codes config', async () => {
+      const actionCodeSettings = {
+        handleCodeInApp: true,
+        url: 'http://localhost:3030/email',
+      };
+
+      const random = Utils.randString(12, '#aA');
+      const email = `${random}@${random}.com`;
+      const pass = random;
+
+      const { user } = await firebase.auth().createUserWithEmailAndPassword(email, pass);
+
+      try {
+        await user.sendEmailVerification(actionCodeSettings);
+        return Promise.reject(new Error('Expected an error with incorrect action codes config'));
+      } catch (e) {
+        /* Update error message */
+        e.message.should.containEql("'options' expected an object value");
+        return Promise.resolve();
```

Comment:
@dackers86  This is odd - so I'm reviewing this, and the changes are 1) add a test that looks for this message and 2) add iOS code that throws an error that has nothing to do with this message

It is like the test is not related to the added iOS code at all? It may be test for some other previously untested chunk of code (excellent if so) but then there is no test for the ios code added? is iOS error 17999 (what we catch and mutate in the new iOS code not testable?